MaplePacketLib2/Tools/PacketWriter.cs

```diff
@@ -180,7 +180,13 @@ public void WriteUnicodeString(string value = "")
 
     public void WriteRawUnicodeString(string value)
     {
+        if (value.Length == 0)
```

Comment:
Why can't we just make the default an empty string instead?

Reply:
Can you clarify what you mean please?

Reply:
As in, why can't we allow blank strings to "write"? does it actually error out or something?

Reply:
The caller already writes a string length of 0 and the function's expected behavior seems to be "write nothing" if the string is empty. The EnsureCapacity call in this function also fatally fails if the string length is 0 which is why this guard was put in.